core/src/main/java/com/scalar/db/api/ScanBuilder.java

```diff
@@ -1221,13 +1207,15 @@ public BuildableScanFromExistingWithOngoingWhereOr whereOr(
     public BuildableScanOrScanAllFromExisting clearStart() {
       checkNotScanWithIndexOrScanAll();
       this.startClusteringKey = null;
+      this.startInclusive = false;
```

Comment:
This could be a (very minor) compatibility issue for those who expect this method to keep the inclusion setting? I think this is a reasonable bugfix, though.

Reply:
Strictly speaking, it could be. But we basically don't use `startInclusive` if `startClusteringKey` is null, so  this shouldn't cause any issues actually. Thanks.